lab-dustin/server.js

```diff
@@ -17,5 +17,5 @@ if (!module.parent){
     console.log(`Listening to port ${PORT}`);
   });
 }
-
+//commit to open the branch!
```

Comment:
For future reference, you don't need to actually commit to make a new branch. You can branch from anywhere.